plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/io/FileDownloader.java

```diff
@@ -86,8 +85,8 @@ public IPath download(URL url, IProgressMonitor monitor) throws IOException {
     connection.setReadTimeout(DEFAULT_READ_TIMEOUT_MS);
     connection.setRequestProperty("User-Agent", CloudToolsInfo.USER_AGENT);
     try (InputStream inputStream = new BufferedInputStream(connection.getInputStream());
-         OutputStream outputStream =
-             new BufferedOutputStream(new FileOutputStream(downloadedFile))) {
+         OutputStream outputStream = 
+             new BufferedOutputStream(Files.newOutputStream(downloadedFile.toPath()))) {
```

Comment:
That doesn't sound right to me. The byte code should be the same either way. Do you have a reference?

Also note that the buffered stream constructors do not throw exceptions.